### PR TITLE
Integration Tests for Sink Implementation

### PR DESCRIPTION
**Pull request by prashastia**
1. Removed all previous read tests as write also performs read.
2. Modified the `BigQueryIntegrationTest.jar` to take destination project, dataset and table name as information.
3. Adds python files to check the total row count and unique key count (`assert_table_count.py`).
* [Link to Cloud-Build](https://pantheon.corp.google.com/cloud-build/builds;region=global/39ba9281-730a-4479-81f1-7c2fa872e551;step=8?project=testproject-398714) Which is a successfull execution of the changes added.
/gcbrun

### CODE REVIEW

File: cloudbuild/nightly/scripts/python-scripts/requirements.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 absl-py==2.0.0
-avro==1.11.1
```
Comment (prashastia) [inline cloudbuild/nightly/scripts/python-scripts/requirements.txt:2]: To fix PR [#90](https://github.com/GoogleCloudDataproc/flink-bigquery-connector/pull/90)

File: cloudbuild/nightly/scripts/python-scripts/requirements.txt [hunk 2]
```diff
@@ -19,7 +19,7 @@ grpc-google-iam-v1==0.12.7
 grpcio==1.59.3
 grpcio-status==1.59.3
 httplib2==0.22.0
-idna==3.6
```
Comment (prashastia) [inline cloudbuild/nightly/scripts/python-scripts/requirements.txt:22]: To fix PR [#106](https://github.com/GoogleCloudDataproc/flink-bigquery-connector/pull/106)

File: cloudbuild/nightly/scripts/bounded_table_write.sh [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+PROPERTIES=$1
+BOUNDED_JOB_SINK_PARALLELISM=$2
+
+# We won't run this async as we can wait for a bounded job to succeed or fail.
+# Create the destination table from the source table schema.
+python3 cloudbuild/nightly/scripts/python-scripts/create_sink_table.py -- --project_name "$PROJECT_NAME" --dataset_name "$DATASET_NAME" --source_table_name "$SOURCE_TABLE_NAME" --destination_table_name "$DESTINATION_TABLE_NAME"
+# Set the expiration time to 1 hour.
+bq update --expiration 3600 "$DATASET_NAME"."$DESTINATION_TABLE_NAME"
```
Comment (jayehwhyehentee) [inline cloudbuild/nightly/scripts/bounded_table_write.sh:23]: Great!

**Review by jayehwhyehentee — APPROVED** [review]
LGTM 👍

**Pull request merged**